.travis.yml

```diff
@@ -19,6 +19,7 @@ script:
   - echo "quit" | docker-compose run web rails dbconsole
   - docker-compose run web rubocop -R --config .rubocop_todo.yml
   - docker-compose run web rake db:drop db:create db:migrate db:seed
+  - gem install brakeman && brakeman --no-pager || true
```

Comment:
the `gem install` should do in the install section. I think that improves the caching, but I'm not 100% sure of that.